esx_service/utils/kvESX.py

```diff
@@ -175,24 +185,28 @@ def load(volpath):
     except IOError:
         return None
 
-    kvDict = json.load(fh)
-
+    kv_str = fh.read()
     fh.close()
+    try:
+       kv_dict = json.loads(kv_str)
```

Comment:
let me rephrase. Why catch here (where we just return None to be taken care upstairs) instead of catching upstairs and do whatever upstairs is doing on "none" result ?